### PR TITLE
Detect if the invoke command is running with PyApp

### PR DESCRIPTION
**Pull request by FlorentClarret**
### What does this PR do?

Detect if the invoke command is running with [PyApp](https://datadoghq.atlassian.net/wiki/spaces/~61a7d8f7ebce470067858b06/pages/3732504868/Datadog+Agent+-+Run+your+invoke+tasks+with+devagent)

### Motivation

I'd like to know

### CODE REVIEW

**Review by KevinFairise2 — APPROVED** [review]

**Review by CelianR — APPROVED** [review]

File: tasks/custom_task/custom_task.py [hunk 1]
```diff
@@ -47,6 +47,7 @@ def get_running_modes() -> list[str]:
         "pre_commit": os.environ.get("PRE_COMMIT", 0) == "1",
         "invoke_unit_tests": is_running_ut or os.environ.get("INVOKE_UNIT_TESTS", 0) == "1",
         "ci": running_in_ci(),
+        "pyapp": os.environ.get("PYAPP", 0) == "1",
```
Comment (ofek) [inline tasks/custom_task/custom_task.py:50]: I know this is just to match the preceding lines but the return value will only ever be strings so we should change the default values here eventually.

**Review by ofek — APPROVED** [review]

File: tasks/custom_task/custom_task.py (hunk 1 above)
Comment (FlorentClarret) [reply tasks/custom_task/custom_task.py:50]:
```suggestion
        "pyapp": os.environ.get("PYAPP", "") == "1",
```

Would this work?

Comment (ofek) [reply tasks/custom_task/custom_task.py:50]: Yes that would be fine for all of them.

Comment (FlorentClarret) [reply tasks/custom_task/custom_task.py]: Fixed. Thank you Ofek

**Review by ofek — APPROVED** [review]

**Pull request merged**